nexteuropa_language_selector_site_popup/nexteuropa_language_selector_site_popup.module

```diff
@@ -64,7 +69,8 @@ function nexteuropa_language_selector_site_popup_preprocess_splash(&$variables)
   // Add enabled languages, if not in blacklist. Add all these languages to our
   // processed list.
   foreach ($languages[1] as $lang) {
-    if (!in_array($lang->prefix, $languages_blacklist)) {
+    $node_tranlated = isset($nid) ? _dt_core_is_node_translated($nid, $lang->language) : FALSE;
```

Comment:
@mathiasselleslach, how could we have this in the theme tools? this is referring to a function belonging to dt_core, if that is a function that needs to be shared it has to be moved in one of our tools.